### DISCUSSION TITLE
Tipologia lezione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#99] [Idee e proposte]
Buongiorno ,sarebbe possibile nel momento in cui si va a firmare la lezione scegliere se si tratta di una lezione in presenza (PRE)o in didattica a distanza  (DAD) o meglio ancora in didattica digitale integrata (DDI)?

**Comment by trinko** [comment]
La aggiungo alla richieste di nuove funzionalità.
